api/api/model_endpoint_alerts_api.go

```diff
@@ -47,9 +45,9 @@ func (c *AlertsController) ListModelEndpointAlerts(r *http.Request, vars map[str
 	modelEndpointAlerts, err := c.ModelEndpointAlertService.ListModelAlerts(modelID)
 	if err != nil {
 		if gorm.IsRecordNotFoundError(err) {
-			return NotFound(fmt.Sprintf("ListModelAlerts: Alerts for Model ID %s not found", modelID))
+			return NotFound(fmt.Sprintf("Model alert not found: %v", err))
```

Comment:
The error messages created in the service / api layers will eventually only be logged by the API router. The ID is expected to be covered by the "path" field in the log created by the router (as in the screenshot). But yes, it some scenarios, it could be confusing (eg: when looping through a list of models and one of them has an issue). If such scenarios arise, we could be more explicit about the error messages.

Reply:
Okay, got it! Thanks for clarifying! :)